PowerShellGet/PSGet.Resource.psd1

```diff
@@ -70,6 +70,10 @@ ConvertFrom-StringData @'
         PublishModuleLocation=Module '{0}' was found in '{1}'.
         InvalidModuleToPublish=Module '{0}' cannot be published because it does not have a module manifest file. Run New-ModuleManifest -Path <PathName> to create a module manifest with metadata before publishing.         
         MissingRequiredManifestKeys=Module '{0}' cannot be published because it is missing required metadata. Verify that the module manifest specifies Description and Author.
+        

```

Comment:
I agree, I'll take them out.  I was using the spaces to find my resources within the list easily as I developed.